src/components/v5/shared/Modal/Modal.module.css

```diff
@@ -3,7 +3,7 @@
 }
 
 .modal {
-  @apply relative outline-0 overflow-hidden bg-base-white md:max-h-[calc(100vh-4rem)] md:w-[30.3125rem] md:border md:border-gray-200 md:rounded-xl shadow-default flex flex-col w-full;
+  @apply relative outline-0 overflow-hidden bg-base-white md:h-auto md:max-h-[calc(100vh-4rem)] md:w-[30.3125rem] md:border md:border-gray-200 md:rounded-xl shadow-default flex flex-col w-full;
```

Comment:
I've done some changes in this file so please double check the modal on desktop and mobile if you would have conflict to resolve